src/BaGet.Core/Configuration/BaGetOptions.cs

```diff
@@ -21,6 +21,12 @@ public class BaGetOptions
         /// </summary>
         public bool RunMigrationsAtStartup { get; set; } = true;
 
+        /// <summary>
+        /// If enabled, the packages folder will be enumerated and every package
+        /// is ensured to exist in the database.
+        /// </summary>
+        public bool StartupEnsurePackagesExistsInDatabase { get; set; }
```

Comment:
Let's create a new option `AutomaticallyIndexedFolder` with two properties: `Enabled` and `Path`.